docs/react/tutorial/01-create-application.md

```diff
@@ -23,11 +23,12 @@ Workspace name (e.g., org name)     myorg
 What to create in the new workspace react
 Application name                    todos
 Default stylesheet format           CSS
+Use Nx Cloud? (It's free and doesn't require registration.) No
```

Comment:
I would remove this and leave the choice up to the person going through the tutorial